package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@remoteoss/json-schema-form",
```

Comment:
suggestion: a better name for this PR:

```
fix(fieldset): Support customProperties with sub-fields clashing with reserved words.
```

the scope is just "fieldset" and the the body explains what was added.